Cargo.toml

```diff
@@ -3,6 +3,8 @@ resolver = "2"
 members  = [ "yazi-*" ]
 
 [profile.release]
+codegen-units = 1
+opt-level = 3
```

Comment:
Seems `opt-level = 3` is the default value of releasing, https://doc.rust-lang.org/cargo/reference/profiles.html#release

Reply:
ahh yea.

Reply:
Okay, I'll only keep `codegen-units`.